pkg/logger/logger_test.go

```
@@ -168,3 +175,165 @@ func TestLogger_SetLogLevel(t *testing.T) {
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			// Pipe to capture output
+			r, w, _ := os.Pipe()
+			oldStdout := os.Stdout
+			os.Stdout = w
+
+			// Channel to capture output
+			outC := make(chan string)
+			go func() {
+				var buf bytes.Buffer
+				io.Copy(&buf, r)
```

golangci-lint (1.62.2)

Error return value of `io.Copy` is not checked

(errcheck)